pkgs/by-name/ve/versitygw/package.nix

```
@@ -0,0 +1,36 @@
+{
+  lib,
+  fetchFromGitHub,
+  buildGoModule,
+}:
+
+buildGoModule rec {
+  pname = "versitygw";
+  version = "1.0.8";
+
+  src = fetchFromGitHub {
+    owner = "versity";
+    repo = "versitygw";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-hrGPHl8vfLsL1JMcaU+RAQj6CaBJKsW+Q2AGCbgYUSA=";
+  };
+
+  vendorHash = "sha256-zZufxxZZ5Lfr0vWcygXdiRK+bhUr/+MFk4ajJGz5TMI=";
+
+  doCheck = false;
```

Why? Add reason such as

```suggestion
  doCheck = false; # Require access to online S3 service
```

```suggestion
  doCheck = false; # Require binding to privileged ports
```

Note that you should prefer partially skipping tests whenever possible, see https://nixos.org/manual/nixpkgs/unstable/#ssec-skip-go-tests